app/controllers/claims/pages_controller.rb

```diff
@@ -1,2 +1,5 @@
 class Claims::PagesController < ApplicationController
+  def index
+    @school = GiasSchool.first
```

Comment:
I think this should be `current_user.schools`

Reply:
My take on this ticket is that it is mainly a "front end" ticket so i tried to not touch business logic and the association between schools and user being absent I decided to use static data. I guess we could decide to add the business logic here and increase the size of the PR or to have a second ticket. I don't mind

Reply:
Perhaps it's something to discuss as a group -- we don't have properly written up tickets / ACs / requirements, so it's difficult to know where to draw the lines. My view is that seeing the relevant school is key to the feature, so should be implemented here.